drivers/video/Kconfig.sw_generator

```diff
@@ -6,7 +6,7 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-menuconfig VIDEO_SW_GENERATOR
-        bool "Video Software Generator"
-        help
-          Enable video pattern generator (for testing purpose).
+config VIDEO_SW_GENERATOR
```

Comment:
was this meant to go from menuconfig -> config?  If so maybe doing that in a separate commit.